src/components/Loader/Loader.scss

```diff
@@ -0,0 +1,25 @@
+.Loader {
+  display: flex;
+  width: 100%;
+  justify-content: center;
+  align-items: center;
+
+  &__content {
```

Comment:
Consider adding a brief comment to describe the purpose of the '__content' class. This will make the code easier to understand for future developers. 😊